packages/cactus-plugin-ledger-connector-polkadot/src/test/rust/fixtures/ink!/publicBulletin/README.md

```diff
@@ -0,0 +1,19 @@
+## Table of Contents<!-- omit in toc -->
```

Comment:
@RafaelAPB Please remove uppercase charachters from the directory names. Same goes for special characters like the exclamation mark.